implementations/js-ajv-plus/bowtie_ajv_plus.js

```diff
@@ -0,0 +1,128 @@
+const path = require("path");
+const readline = require("readline");
+const os = require("os");
+const process = require("process");
+
+const ajv_version = require(
+  path.join(
+    path.dirname(path.dirname(require.resolve("@theflashlabs/ajv-plus"))),
+    "package.json",
+  ),
+).version;
+
+const DRAFTS = {
+  "https://json-schema.org/draft/2020-12/schema": require("@theflashlabs/ajv-plus/dist/2020"),
+  "https://json-schema.org/draft/2019-09/schema": require("@theflashlabs/ajv-plus/dist/2019"),
+  "http://json-schema.org/draft-07/schema#": require("@theflashlabs/ajv-plus"),
+  "http://json-schema.org/draft-04/schema#": require("@theflashlabs/ajv-draft-04"),
+};
+var ajv;
+
+const stdio = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+  terminal: false,
+});
+
+function send(data) {
+  console.log(JSON.stringify(data));
+}
+
+var started = false;
+
+const cmds = {
+  start: (args) => {
+    console.assert(args.version === 1, { args });
+    started = true;
+    return {
+      version: 1,
+      implementation: {
+        language: "javascript",
+        name: "theflashlabs-ajv-plus",
+        version: ajv_version,
+        homepage: "https://github.com/theflashlabs/ajv-plus",
+        documentation: "https://github.com/theflashlabs/ajv-plus",
+        issues: "https://github.com/theflashlabs/ajv-plus/issues",
+        source: "https://github.com/theflashlabs/ajv-plus",
+
+        dialects: [
+          "https://json-schema.org/draft/2020-12/schema",
+          "https://json-schema.org/draft/2019-09/schema",
+          "http://json-schema.org/draft-07/schema#",
+          "http://json-schema.org/draft-06/schema#",
+          "http://json-schema.org/draft-04/schema#",
+        ],
+        os: os.platform(),
+        os_version: os.release(),
+        language_version: process.version,
+      },
+    };
+  },
+
+  dialect: (args) => {
+    console.assert(started, "Not started!");
+
+    // For some reason ajv's process for Draft 6 is different, so split.
+    if (args.dialect !== "http://json-schema.org/draft-06/schema#") {
+      ajv = new DRAFTS[args.dialect]({ strict: false });
+    } else {
+      const Ajv = require("@theflashlabs/ajv-plus");
+      const draft6MetaSchema = require("@theflashlabs/ajv-plus/dist/refs/json-schema-draft-06.json");
+
+      ajv = new Ajv({ strict: false });
```

Comment:
I see you still explicitly passing `strict: false` here. Isn't that the core thing you were saying ajv-plus changes about ajv's defaults?

The harness is meant to check implementations in their default configuration (at least until we ever do #8, in which case we'd also add ajv with strict mode off).